### ISSUE TITLE
Проблема с установкой приложения на андроид 14

### ISSUE THREAD

**Issue by veopot** [veopot/mwx-json#74]
Проблема с установкой приложения на андроид 14

**Comment by LomikXP** [comment]
В чем она заключается?

**Comment by veopot** [comment]
> В чем она заключается?

![Screenshot_2023-12-15-17-00-22-997](https://github.com/veopot/mwx-json/assets/966464/8797fbbb-bf6e-4d9f-b953-66908df23b0f)


Не возможно установить приложение

**Comment by LomikXP** [comment]
А ну у меня пока не было планов поднимать уровень Api android/ иначе потеряем поддержку старых телефонов, либо это нужно будет выпускать несколько версий ПО.
На текущий момент поддерживается версия android 2.3.

**Comment by veopot** [comment]
> А ну у меня пока не было планов поднимать уровень Api android, иначе потеряем поддержку старых телефонов, либо это нужно будет выпускать несколько версий ПО. На текущий момент поддерживается версия android 2.3.

В данный момент знаю 2 человека не могут установить на андроид14 

Но один вроде как установил через жопу так как у него рут есть

**Comment by HuaweiDonor** [comment]
Pixel 8Pro, так же 14 андроид не ставится.
![Screenshot_20231230-180954](https://github.com/veopot/mwx-json/assets/5327926/dab7ff3d-b535-4237-aa53-4c31926b938f)

**Comment by AlexSidney95** [comment]
Nothing phone (1), android 14. Так же не ставится любимая читалка :-(
LomikXP, мы есть, кто с самых древних времён с manga watcher и олдовые устройства на андроид 2.3 уже истлели давно. За всех не скажу, но потребность в поднятии api для новых устройств кажется приоритетней, чем сохранение поддержки старых версий. Да и не зачем две версии ПО поддерживать, прошлые версии программы никуда не денутся, просто станет конечная версия для какой-то версии Андроид, причём уже давно отполированная, это же норма.

**Comment by HuaweiDonor** [comment]
Принял для себя решение уйти на похожее приложение, есть подписка но она не особо дорогая, при этом есть поддержка от разраба

**Comment by AlexSidney95** [comment]
Нет, я предан этому приложению, оно уже почти десять лет у меня в основе

**Comment by grenzentur7** [comment]
Тоже приобрела телефон с андроид 14. Уже давно пользуюсь читалкой, и не хочу переходить ни на что другое, с Manga Watcher ничего не сравниться. Надеюсь и верю, что все таки что нибудь да получится придумать для новых андроидов.

**Comment by Last8Exile** [comment]
Для установки на андроид 14 targetSDK должен быть >= 23. для minSDK ограничений нет. У этого приложения вроде как targetSDK = 26. Такчто должно устанавливаться. Точную причину 
сбоя можно посмотреть устанавливая приложения через adb install.

И у меня это проблема с отсутствием armv8 64 битного кода. Т.е. приложение поддерживает содержит только armv7 32 битный код.

Snapdragon 8 gen 3 не поддерживает armv7

**Comment by veopot** [comment]
Эдем ответа разработчика @LomikXP

**Comment by LomikXP** [comment]
Попробуйте
https://github.com/veopot/mwx-json/blob/main/apk_mangawatcherx/beta/manga-watcher-3.1.6.8-api23.apk

**Comment by veopot** [comment]
@Last8Exile @grenzentur7 @AlexSidney95 
Тестируем у кого андроид 14

**Comment by AlexSidney95** [comment]
С утра гоняю, установилось быстро, вылезло окошко "это приложение для более ранней версии Android блаблабла", далее пока проблем не встретил.

**Comment by Last8Exile** [comment]
Попробовал. Работает.  Окошка несовместимости небыло.

Хотя я и так смог поставить обычную, пропатчив в ней версию minSDK.

**Comment by AlexSidney95** [comment]
Встретил вот такую ошибку, раньше такого не встречал. В данном случае при открытии главы берсерка на com-x.life вылезает окошко с запросом возраста и всё друг об друга спотыкается, кнопки кликабельны, но ничего не происходит.
![Screenshot_20240214-103220](https://github.com/veopot/mwx-json/assets/157781604/b88d8259-9658-4c7d-9dfe-e34ea25b4e67)

**Comment by veopot** [comment]
> Встретил вот такую ошибку, раньше такого не встречал. В данном случае при открытии главы берсерка на com-x.life вылезает окошко с запросом возраста и всё друг об друга спотыкается, кнопки кликабельны, но ничего не происходит. ![Screenshot_20240214-103220](https://private-user-images.githubusercontent.com/157781604/304657041-b88d8259-9658-4c7d-9dfe-e34ea25b4e67.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc5MDA0MzUsIm5iZiI6MTcwNzkwMDEzNSwicGF0aCI6Ii8xNTc3ODE2MDQvMzA0NjU3MDQxLWI4OGQ4MjU5LTk2NTgtNGM3ZC05ZGZlLWUzNGVhMjViNGU2Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxNFQwODQyMTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMjYxN2JjMDY1MzI3NmY2YWVkNTE3Y2RmNTZjNzlhY2VlMTZkMzk5MmVmMTRiMTcxMDUwMWRkZGYzZjA4Njg4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.5innhtA2gbXV9JqhHAMQ_6NR96_mOe4E_qF5aJg8AAM)

Про это уже знаю давно 
Вот тема на эту проблему 
https://github.com/veopot/mwx-json/issues/52

**Comment by Miniromano** [comment]
Чёт меня чорт за ногу дёрнул на новую версию обновится, и ничего встретился с ошибкой которые специальной версией фиксится однако я теперь не могу обновить библиотеку. Пишет "библиотека обновляется попробуйте позже" 
И я теперь я не получаю обновления глав нужно входит вручную и обновлять у добавленной манги.
![Screenshot_20241227_015252_Manga Watcher X](https://github.com/user-attachments/assets/8cecba4e-7f0d-4f5f-b4e6-ea21e8a6d92a)
![Screenshot_20241227_015347_Manga Watcher X](https://github.com/user-attachments/assets/26365881-a22c-4ab4-a87b-f6f94d6f429b)

Телефон: Samsung S24 Ultra  
Версия Андроида 14 

Пользуюсь по памяти с 2016 года и очень не хотелось бы терять эту читалку.

**Comment by LomikXP** [comment]
Пока не понятно, пробуйте последние версии с суффиксом -actual

**Comment by veopot** [comment]
> Чёт меня чорт за ногу дёрнул на новую версию обновится, и ничего встретился с ошибкой которые специальной версией фиксится однако я теперь не могу обновить библиотеку. Пишет "библиотека обновляется попробуйте позже" И я теперь не получаю обновления глав нужно входит вручную и обновлять у добавленной манги. ![Screenshot_20241227_015252_Manga Watcher X](https://github.com/user-attachments/assets/8cecba4e-7f0d-4f5f-b4e6-ea21e8a6d92a) ![Screenshot_20241227_015347_Manga Watcher X](https://github.com/user-attachments/assets/26365881-a22c-4ab4-a87b-f6f94d6f429b)
> 
> Телефон: Samsung S24 Ultra Версия Андроида 14
> 
> Пользуюсь по памяти с 2016 года и очень не хотелось бы терять эту читалку.

Проверьте разрешены ли у приложение 
В настройках уведомления. Может и за этого не работать 

![Image](https://github.com/user-attachments/assets/8afd611f-cf57-477b-89fa-f81f36aa1963)

**Comment by leshasmlesha** [comment]
https://github.com/veopot/mwx-json/releases/tag/3.1.7.5
